ember-cli-build.js

```diff
@@ -34,6 +35,11 @@ module.exports = function() {
   vendorTrees.push(funnel(path.dirname(require.resolve('handlebars/package')), {
       include: ['dist/handlebars.amd.js'] }));
 
+  if (isTest) {
+    vendorTrees.push(packageDist('@glimmer/application'));
+    vendorTrees.push(packageDist('@glimmer/application-test-helpers'));
```

Comment:
Good call. I just pushed a commit to make the `vendorTrees` and `external` arrays more DRY.